assignment5/src/components/Header.js

```diff
@@ -0,0 +1,19 @@
+import { BiUserCircle } from "react-icons/bi";
+import SearchBar from "./SearchBar";
+
+const Header = (props) => {
+  const { setDeveloperData, userData } = props;
+  return (
+    <div className="header">
+      <div className="logo">Storm Troopers</div>
+      <SearchBar setDeveloperData={setDeveloperData} userData={userData} />
```

Comment:
Here also, userData can be imported directly from teamMatesData.json
I think there is no need to pass userData as props